addons/event_sms/data/sms_data.xml

```diff
@@ -11,7 +11,8 @@
     <record id="sms_template_data_event_reminder" model="sms.template">
         <field name="name">Event: Reminder</field>
         <field name="model_id" ref="event.model_event_registration"/>
-        <field name="body">{{ object.event_id.organizer_id.name or object.event_id.company_id.name or user.env.company.name }}: We are excited to remind you that the {{ object.event_id.name }} event is starting {{ object.get_date_range_str() }}. We confirm your registration and hope to meet you there.</field>
+        <field name="body">Ready for "{{ object.event_id.name }}" {{ object.get_date_range_str(object.partner_id.lang) }}?
+{{ 'It starts at ' + format_time(time=object.event_begin_date, tz=object.event_id.date_tz, time_format='short', lang_code=object.partner_id.lang) + (', at ' + object.event_id.address_id.contact_address if object.event_id.address_id.contact_address else '') + '. See you there!'}}</field>
```

Comment:
Can't you extract python into ``{{...}}`` marks and remove string concatenation "+" signs ?